BLUESPAWN-client/src/hunt/hunts/HuntT1036.cpp

```diff
@@ -0,0 +1,61 @@
+#include "hunt/hunts/HuntT1036.h"
+
+#include "util/log/Log.h"
+#include "util/filesystem/FileSystem.h"
+
+namespace Hunts {
+
+	HuntT1036::HuntT1036() : Hunt(L"T1036 - Masquerading") {
+		dwSupportedScans = (DWORD) Aggressiveness::Cursory;
+		dwCategoriesAffected = (DWORD) Category::Files;
+		dwSourcesInvolved = (DWORD) DataSource::FileSystem;
+		dwTacticsUsed = (DWORD) Tactic::DefenseEvasion;
+	}
+
+	int HuntT1036::ScanCursory(const Scope& scope, Reaction reaction){
+		LOG_INFO("Hunting for " << name << " at level Cursory");
+		reaction.BeginHunt(GET_INFO());
+
+		int detections = 0;
+
+		FileSystem::FileSearchAttribs searchFilters;
+		searchFilters.extensions = susExts;
+
+		for (auto folder : writableFolders) {
+			auto f = FileSystem::Folder(folder);
+			if (f.GetFolderExists()) {
+				LOG_VERBOSE(1, L"Scanning " << f.GetFolderPath());
+				for (auto value : f.GetFiles(searchFilters, -1)) {
+					if (value.GetFileAttribs().extension == L".exe" || value.GetFileAttribs().extension == L".dll") {
```

Comment:
We need a more unified method of identifying PE32+ files

Reply:
should be looking at the file's magic bytes instead of using an extension probably

Reply:
No, the file is executed. We need to parse HKEY_CLASSES_ROOT to determine what will happen.